deployment/ccip/changeset/test_helpers.go

```diff
@@ -1221,6 +1221,73 @@ func Transfer(
 	return msgSentEvent, startBlocks
 }
 
+type TestTransferRequest struct {
+	Name                   string
+	SourceChain, DestChain uint64
+	Receiver               common.Address
+	ExpectedStatus         int
+	// optional
+	Tokens                []router.ClientEVMTokenAmount
+	Data                  []byte
+	ExtraArgs             []byte
+	ExpectedTokenBalances map[common.Address]*big.Int
+}
+
+func TransferMultiple(
+	ctx context.Context,
+	t *testing.T,
+	env deployment.Environment,
+	state CCIPOnChainState,
+	requests []TestTransferRequest,
+) (
+	map[uint64]*uint64,
+	map[SourceDestPair]cciptypes.SeqNumRange,
+	map[SourceDestPair]map[uint64]int,
+	map[uint64]map[TokenReceiverIdentifier]*big.Int,
+) {
+	startBlocks := make(map[uint64]*uint64)
+	expectedSeqNums := make(map[SourceDestPair]cciptypes.SeqNumRange)
+	expectedExecutionStates := make(map[SourceDestPair]map[uint64]int)
+	expectedTokenBalances := make(map[uint64]map[TokenReceiverIdentifier]*big.Int)
+
+	for _, tt := range requests {
+		t.Run(tt.Name, func(t *testing.T) {
+			expectedTokenBalances = AppendExpectedTokenBalances(
```

Comment:
is it possible to run these in parallel?

Reply:
Tests? Tests only send a message and wait for confirmations, these actions are relatively fast compared to other parts of the tests. Parallelizing here is also tricky, because you can parallelize only per chain or per sender. IMO not worth the effort at this stage